src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/ApplicationsOrchestrator.cs

```diff
@@ -100,7 +106,26 @@ public async Task<ApplicationViewModel> GetApplication(ApplicationRequest reques
                  Status = result.Status,
                  EncodedOpportunityId = encodedOpportunityId,
                  EstimatedTotalCost = estimatedTotalCost,
+                 CanAcceptFunding = isOwnerOrTransactor && result.Status == ApplicationStatus.Approved
             };
         }
+
+        public async Task AcceptFunding(AcceptFundingPostRequest request, CancellationToken cancellationToken = default)
```

Comment:
When Chris was doing the sender side of this (accepting/rejecting an application) he made one execution path (orchestrator method, service method etc.) for both accepting and rejecting the application. I.e. in the PledgesOrchestrator the equivalent method here is called SetApplicationOutcome and the SetApplicationOutcomeRequest has an 'Outcome' enum which denotes either Accept or Reject. That way when we add the Reject functionality all we have to do is call the same execution path but with the Outcome set to Reject instead, and the rest will be handled by the already existing functionality. Might be worth checking with Chris if that's what he'd want here.